react/states_and_effects/project_cv_application.md

```diff
@@ -31,15 +31,15 @@ Not only do some of these get rid of additional steps after pushing our code to
 While there are many ways to deploy to Netlify, like uploading your `dist` directly or using [Netlify's `netlify-cli`](https://docs.netlify.com/cli/get-started/) CLI, the most convenient way would be to directly import your GitHub repository to Netlify.
 
 1. Push your React application to GitHub.
-1. [Import your project](https://app.netlify.com/start) to Netlify by logging in, and selecting your repository.
+1. Import your project to [Netlify](https://app.netlify.com/start) by logging in, and selecting your repository.
```

Comment:
```suggestion
1. [Import your project to Netlify](https://app.netlify.com/start) by logging in, and selecting your repository.
```
This ought to include the "import your project to" part, otherwise a screenreader will just read "netlify - link", which isn't fully accurate. That would more likely imply a homepage or general link to the website, whereas we want to specify that the link is specifically to where you start importing a project